test/Microsoft.DocAsCode.Build.Engine.Tests/DocumentBuilderTest.cs

```diff
@@ -960,6 +960,40 @@ public void AddFakeResponse(Uri uri, HttpResponseMessage responseMessage)
             Assert.Equal("/docfx/guideline/csharp_coding_standards.html?x=y", result.Href);
         }
 
+        [Fact]
+        public async Task TestBuildWithXrefServiceRemoveHostWithParameters()
+        {
+            var fakeResponseHandler = new FakeResponseHandler();
+            var httpClient = new HttpClient(fakeResponseHandler);
+
+            fakeResponseHandler.AddFakeResponse(new Uri("http://example.org/test1"), new HttpResponseMessage
+            {
+                StatusCode = HttpStatusCode.OK,
+                Content = new StringContent("[{'uid':'csharp_coding_standards', 'name':'C# Coding Standards', 'href':'http://dotnet.github.io/docfx/guideline/csharp_coding_standards.html'}]")
```

Comment:
Is it supported to remove `dotnet.github.io/docfx`?

Reply:
no, only host name.